spec/system/candidate_interface/apply_again/candidate_reviews_rejection_reasons_from_previous_applications_spec.rb

```diff
@@ -52,6 +52,12 @@ def when_i_apply_again
 
     click_link 'Choose your course'
     candidate_fills_in_apply_again_course_choice
+
+    if FeatureFlag.active?(:reference_selection)
+      click_link 'Selected references'
+      choose 'Yes, I have completed this section'
+      click_button t('save_and_continue')
```

Comment:
Is it worth asserting that the section is displayed as completed maybe in `and_i_can_see_my_referees` below?

Reply:
We're likely going to revisit this, one of the remaining tasks is deciding exactly what happens to selections during apply again.